xml/dashboard_ganesha.xml

```diff
@@ -96,21 +92,18 @@
    </step>
    <step>
     <para>
-     Enter the directory path for the NFS export. If the directory does not
-     exist on the server, it will be created.
+     Enter the directory path for the NFS export.
+     If the directory does not exist on the server, it will be created.
     </para>
    </step>
    <step>
     <para>
-     Specify other NFS related options, such as supported NFS protocol version,
-     pseudo, access type, squashing, or transport protocol.
+     Specify other NFS related options, such as supported NFS protocol version, pseudo, access type, squashing, or transport protocol.
     </para>
    </step>
    <step>
     <para>
-     If you need to limit access to specific clients only, click <guimenu>Add
-     clients</guimenu> and add their IP addresses together with access type and
-     squashing options.
+     If you need to limit access to specific clients only, click <guimenu>Add clients</guimenu> and add their IP addresses together with access type and squashing options.
```

Comment:
Why the removal of line-wrapping? This file now contains many very long lines, which makes viewing them in Github difficult...

Reply:
do you think 1 sentence per line is a bed idea? To me, it has 2 advantages:
- it immediately tells you which sentences are too long
- removing or adding a sentence, or making changes inside a sentence makes the diff easier to read actually, because no auto-line wrapping occurs when sentence become shorter or longer